main.c

```diff
@@ -720,6 +720,8 @@ main(int argc, char **argv)
 		}
 	    }
 #endif
+	    else if (!strcmp("-auto-inline-img", argv[i]))
```

Comment:
Does this need to have a command line switch? It's should be an option anyways and the can be set via -o.

Reply:
I considered that, but thought that `inline_img_protocol=5` or `enable_inline_image == 5` would imply a built-in protocol rather than use of w3mimgdisplay for any code looking for a positive value to mean that, while auto-detection can resolve to 0 (ie. use w3mimgdisplay). Also, if a new image protocol were to become relevant, such as Terminology's, having auto-detection in the middle of a list of protocols would make no sense. Trouble is, set_param() doesn't allow negative values for P_INT and I thought changing that could have excessively far-reaching implications for anything assuming `P_INT` was >= 0.

Looking at set_param again, this could fixed by making auto_inline_image with P_SHORT or P_CHARINT and changing the type of `enable_inline_image` accordingly, but at the time I just went for the easiest and simplest way so I could test it